app/shared/src/commonMain/kotlin/ui/subject/episode/EpisodeVideo.kt

```diff
@@ -251,8 +254,9 @@ internal fun EpisodeVideoImpl(
                     MediaProgressIndicatorText(progressSliderState)
                 },
                 progressSlider = {
+                    val chapters by playerState.chapters.collectAsState()
```

Comment:
```suggestion
                    val chapters by playerState.chapters.collectAsStateWithLifecycle()
```

在 UI 几乎总是要使用 `collectAsStateWithLifecycle` 而不是 `collectAsState`